google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPoolOptions.java

```diff
@@ -190,6 +195,10 @@ boolean isFailOnSessionLeak() {
     return actionOnSessionLeak == ActionOnSessionLeak.FAIL;
   }
 
+  public boolean isTrackStackTraceOfSessionCheckout() {
```

Comment:
I don't think we should, as it would not be in line with (most of) the other public options. The setter is public, so I think it makes sense to allow users to also get the value of the option.